### PR TITLE
basic file structure and readme setup

### PR DESCRIPTION
**Pull request by Arwask**
- just the bare backbone folder structure
- basic package.json
- updated Readme.md 
- setup in package.json to reset/set the database as and when required.
- env settings

### CODE REVIEW

**Review by thejonroberts — CHANGES_REQUESTED** [review]
The folders don't get pushed up unless there is a file inside them.  Could you make at least one in each folder you have?

File: package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Read-Only-API",
+  "version": "1.0.0",
+  "description": "Bangazon-corp's employees, products and users database",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (megbrown) [inline package.json]: Should we go ahead and add ```mocha --recursive --watch -R spec```?

Comment (Arwask) [reply package.json]: It will change for mac and pc users a little bit, but I will go ahead and add it

**Review by megbrown — APPROVED** [review]
Saw the changes. Looks good!

**Review by thejonroberts — APPROVED** [review]
Thanks!

**Review by SamBDev — APPROVED** [review]
fetched the branch and it looks good

**Pull request merged**